lib/src/signed_video_tlv.c

```diff
@@ -236,7 +236,9 @@ encode_general(signed_video_t *self, uint8_t *data)
   if (gop_info->has_timestamp) {
     data_size += sizeof(timestamp);
   }
-
+  if (self->linked_hash_on) {
```

Comment:
As mentioned above. There is a misunderstanding of usage of the new linked hash. It should always be written. Remove this if.